lib/node_modules/@stdlib/array/base/mskfilter-map/benchmark/benchmark.length.js

```diff
@@ -0,0 +1,98 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var zeroTo = require( '@stdlib/array/base/zero-to' );
+var ones = require( '@stdlib/array/base/ones' );
+var isArray = require( '@stdlib/assert/is-array' );
+var identity = require( '@stdlib/math/base/special/identity' );
+var pkg = require( './../package.json' ).name;
+var mskfiltermap = require( './../lib' );
```

Comment:
```suggestion
var mskfilterMap = require( './../lib' );
```

Consistency with elsewhere.